libraries/c_sdk/standard/mqtt/src/iot_mqtt_subscription_container.c

```diff
@@ -69,7 +69,7 @@ static bool _topicMatch( _mqttSubscription_t * pSubscription,
 /*-----------------------------------------------------------*/
 
 static bool _packetMatch( _mqttSubscription_t * pSubscription,
-                          void * pMatch )
+                          const void * pMatch )
```

Comment:
why to remove const qualifier?

Reply:
I have added the const qualifier as the matching params should not change.